src/lib/remark-plugins/rewrite-tutorial-links/__tests__/rewrite-external-docs-link.test.ts

```diff
@@ -6,14 +6,23 @@ const getTestURLObject = (url: string) => {
 	return new URL(url, __config.dev_dot.canonical_base_url)
 }
 
+const URL_SUFFIXES = ['', '?param=value', '#heading', '?param=value#heading']
```

Comment:
A note would be good here as it makes the test a bit harder to see at first glance since none of the assertions talk about the hash / query param inputs / expectations.

Reply:
💭 Rather than having the `testEachCase` always add all suffixes, it could be more clear to separate this into its own function, so that we can read the intent. So there would be a base `testEachCase` that doesn't append or modify the paths, and a `testEachCaseWithParams` — to show that there's a manipulation happening with those paths.

Reply:
That said, I think a note would totally suffice as well 😄 its up to you